docs/DeveloperGuide.md

```diff
@@ -257,33 +257,46 @@ _{Explain here how the data archiving feature will be implemented}_
 
 **Target user profile**:
 
-* has a need to manage a significant number of contacts
+* has a need to contact and liaise with many clients to sell their products
 * prefer desktop apps over other types
 * can type fast
 * prefers typing to mouse interactions
 * is reasonably comfortable using CLI apps
+* split clients according to potential and secured
+* has many different types of clients (etc. high/low accident rate, injury-prone)
+* has difficulty remembering clients' information
+* faces trouble in scheduling client meet-ups for both time and location
+* wish to filter based on address
 
-**Value proposition**: manage contacts faster than a typical mouse/GUI driven app
+**Value proposition**: 
+* manage clients faster than a typical mouse/GUI driven app
+* categorise client according to status and risk for the user to plan their schedule
+* inbuilt calendar to track meetings
```

Comment:
Not sure about this calendar. Hopefully we don't have to build a whole UI. 🤞 

We can def keep track of / sort meeting times tho.